design/cassandra/loss-of-persistence.md

```diff
@@ -0,0 +1,69 @@
+# Loss of Persistence - The Case of Local Disks
+
+## The Problem
+
+* Network Attached Storage is fault tolerant but slower and more expensive.
+* Local Storage is much faster and cheaper but it is tied to the Node it lives on.
+
+NoSQL databases like Cassandra implement fault tolerance and were designed with hardware failure in mind.
+We should leverage these capabilities in the Cassandra Operator, 
+so users can easily take advantage of the vastly superior performance of local disks.
+
+These fault tolerant capabilities useful for all kinds of storage in case of data loss, 
+but they are especially important for local storage, where failures are expected to happen often.
+
+## Background
+
+### Kubernetes Support For Local Storage
+
+From the beginning of Kubernetes, storage solutions have focused on Network Attached Storage.
+There has been very little support for using local storage and the way to do it until recently was by using hostPath volumes, which had a multitude of problems.
+
+With Kubernetes 1.10, [the local Persistent Volumes feature graduated to beta](https://kubernetes.io/blog/2018/04/13/local-persistent-volumes-beta/), making local storage portable and easy to use on Kubernetes.
+In order to use a local disk as a Persistent Volume, users can leverage [the local volume static provisioner](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner). 
+To use a local Persistent Volume for a Cassandra Cluster, the user only needs to specify the correct `storageClassName`.
```

Comment:
since only a static provisioner is available for local storage currently, am i correct in my understanding that this requires the admin to manually create/provision these local volumes before they can be passed along to the cassandra operator?

Reply:
Yes, the admin needs to pre-provision local volumes.
The admin then deploys the static provisioner and defines in its configuration, the paths under which to find the local disks.
After that, the static provisioner automates the lifecycle management of the local PVs.